__tests__/app.test.js

```diff
@@ -113,14 +113,98 @@ describe('GET /api/articles', () => {
       expect(body.articles).toEqual(sortedArray);
     });
   });
-  test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
+  test('GET:200 sends all articles when no topic is specified with correct properties and data types', () => {
+    return request(app).get('/api/articles').expect(200).then(({ body }) => {
+      expect(body.articles).toBeInstanceOf(Array);
+      expect(body.articles.length === 13).toBe(true);
+      expect(body.articles).toBeInstanceOf(Array);
+
+      body.articles.forEach((article) => {
+        expect(article).toBeInstanceOf(Object);
+
+        expect(article).toHaveProperty('article_id');
+        expect(article).toHaveProperty('title');
+        expect(article).toHaveProperty('author');
+        expect(article).toHaveProperty('topic');
+        expect(article).toHaveProperty('created_at');
+        expect(article).toHaveProperty('votes');
+        expect(article).toHaveProperty('article_img_url');
+        expect(article).toHaveProperty('comment_count');
+
+        expect(typeof article.article_id).toBe('number');
+        expect(typeof article.title).toBe('string');
+        expect(typeof article.author).toBe('string');
+        expect(typeof article.topic).toBe('string');
+        expect(typeof article.created_at).toBe('string');
+        expect(typeof article.votes).toBe('number');
+        expect(typeof article.article_img_url).toBe('string');
+        expect(typeof article.comment_count).toBe('number');
+      });
+    });
+  });
+  test('GET:200 sends all articles when no topic is specified with correct data', () => {
+    return request(app).get('/api/articles').expect(200).then(({ body }) => {
+      expect(body.articles[0]).toHaveProperty('article_id', 3);
+      expect(body.articles[0]).toHaveProperty('title', 'Eight pug gifs that remind me of mitch');
+      expect(body.articles[0]).toHaveProperty('author', 'icellusedkars');
+      expect(body.articles[0]).toHaveProperty('topic', 'mitch');
+      expect(body.articles[0]).toHaveProperty('created_at', '2020-11-03T09:12:00.000Z');
+      expect(body.articles[0]).toHaveProperty('votes', 0);
+      expect(body.articles[0])
+        .toHaveProperty('article_img_url', 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+      expect(body.articles[0]).toHaveProperty('comment_count', 2);
+    });
+  });
```

Comment:
^^ I'm not sure these last two tests are necessary! Task 5 will have covered these tests :)